src/control/IntersectionReferenceSystem.ts

```diff
@@ -102,7 +111,14 @@ export class IntersectionReferenceSystem {
    */
   project(length: number): number[] {
     const { curtain } = this.interpolators;
-    const l = (length - this._offset) / this.length;
+    const { calculateDisplacementFromBottom } = this.options;
+
+    let l: number;
+    if (calculateDisplacementFromBottom) {
+      l = (this.length - (length - this._offset)) / this.length;
+    } else {
+      l = (length - this._offset) / this.length;
+    }
```

Comment:
Can't help but thinking one calculates a range from [0, 1] and the other [1, 0], the two calculations look very similar. Too late on friday for an elegant solution
l = (length - this._offset) / this.length;
if (calculateDisplacementFromBottom) {
   l = 1-l;
}

Reply:
Ficed